@navikt/ds-react/src/link/Link.tsx

```diff
@@ -3,30 +3,13 @@ import cl from "classnames";
 import { OverridableComponent } from "..";
 
 export interface LinkProps
-  extends React.AnchorHTMLAttributes<HTMLAnchorElement> {
-  /**
-   * Link text
-   */
-  children: React.ReactNode;
```

Comment:
Hva tenker du å gjøre så children ikke lenger er required?

Reply:
Ble vi enige om å bare beholde children der vi tenker det er relevant for komponenten?